docker-compose.yml

```diff
@@ -21,4 +19,4 @@ services:
       MONGO_INITDB_ROOT_USERNAME: $MONGODB_ADMIN_USER
       MONGO_INITDB_ROOT_PASSWORD: $MONGODB_ADMIN_PASSWORD
     volumes:
-      - <path-to-your-/data/db>:/data/db
+      - $MONGO_DATA_PATH:/data/db
```

Comment:
Does this have to be a path? I'm using a Docker managed volume currently

Reply:
@JackVCurtis I'm thinking on how to make this easier for someone to set up. I imagine we would need a mongodump of the data and import it. Was your volume set up before running mongo in docker..?

Reply:
Yeah I ran the mongo container independent of docker-compose to import the data dump.

Reply:
👍 Thanks for that. Made the change f1f0b7481f8ed6ba80418e19b6c66830bc2009a4